_includes/header.html

```diff
@@ -1,8 +1,35 @@
 <header class="fx-row fx-row-between-xs fx-text-center">
-  <div class="fx-col fx-text-center"><a href="/{% if site.active_lang != site.default_lang %}{{ site.active_lang }}/{% endif %}">
-      <img alt="{{ site.data.general.nav.altJamulusIcon }}" width="40" height="40" src="{{ 'assets/img/jamulus-icon-2020.svg'| relative_url }}" /></a>
+  <div class="fx-col fx-text-center icon">
+    <a href="/{% if site.active_lang != site.default_lang %}{{ site.active_lang }}/{% endif %}">
+        <img alt="{{ site.data.general.nav.altJamulusIcon }}" width="40" height="40" src="{{ 'assets/img/jamulus-icon-2020.svg'| relative_url }}" />
+        <span>
+          <div>Jamulus</div>
+          <div class="tagline">{{ site.data.general.header.tagline }}</div>
+        </span>
+    </a> 
+      </div>
+        
+  </div>
+  <div class="fx-col-center-xs fx-txt-right headerlinks">
+    <a href="/wiki/Getting-Started">{{ site.data.general.header.setupLink }}</a>
+    <a href="/wiki/Software-Manual">{{ site.data.general.header.documentationLink }}</a>
+    <a class="download-header" href="/Download">{{ site.data.general.header.downloadLink }}</a>
   </div>
   <div class="fx-col-center-xs fx-txt-right hide-on-dt">
     <a href="#mnv" onclick="showNav();return false;">{% include general/navbtn.html %}</a>
   </div>
 </header>
+
+<!-- highlights current page in navbar -->
+<script>
```

Comment:
Can this be done with CSS only?

Reply:
Not entirely sure, you might be able to, but it'd probably be much more cumbersome. Is there a better place to add scripts?

Reply:
Yes. in assets/js/

Nevertheless: why don't you add a class via jekyll logic? See the navigation bar which includes an active class for pages you're on.

Reply:
I'll give this a go, I think that should work

Reply:
See: https://github.com/jamulussoftware/jamuluswebsite/blob/3431f8315324ee0edc094447ce8b8cd38f26c4a8/_includes/general/nav.html#L9

Reply:
Great, I got it working - thanks